builder/parallels/common/step_attach_cd.go

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package common
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+)
+
+// StepAttachCD is a step that attaches a cd to the virtual machine.
+//
+// Uses:
+//
+//	driver Driver
+//	ui packersdk.Ui
+//	vmName string
+//
+// Produces:
+type StepAttachCD struct {
+	cdRomType   string `mapstructure:"cdrom_type"`
```

Comment:
what's the use of mapstructure here ? Remove this too.

Reply:
Done.